kedro/runner/thread_runner.py

```diff
@@ -147,6 +147,10 @@ def _run(  # pylint: disable=too-many-locals,useless-suppression
                         self._suggest_resume_scenario(pipeline, done_nodes)
                         raise
                     done_nodes.add(node)
+                    self._logger.info("Completed node: %s", node)
```

Comment:
Wouldn't this be more readable? 

```suggestion
                    self._logger.info("Completed node: %s", node.name)
```

Reply:
Indeed, ha I never noticed that it's so verbose throughout the project 🙈